content/monitors/faq/as-count-monitor-evaluation.md

```diff
@@ -0,0 +1,104 @@
+---
+title: as_count() monitor evaluation changes
+kind: faq
+---
+
+## Overview
+
+Currently monitors that have the `as_count` modifier use a separate evaluation path than other monitors. In certain use-cases this this can result in unexpected behavior. We intend to migrate all monitors with `as_count` to the same evaluation path as other monitors, but this doc explains the underlying issue and will guide you through the migration process.
+
+Let's call the current evaluation path for `as_count` monitors `current_eva_path` and the new one `new_eva_path`.  
+
+The difference between those two evaluation paths is that complex monitor queries, especially those that have division or multiplication, may produce **unintended results** depending on the time aggregation function:
```

Comment:
Only division. In case of `*`, `+` or `-` the order of application of time aggregation function does not matter.

Reply:
You should also mention that `pct_change` function is same as `a - timeshift(a, x) / timeshift(a, x)` which carries division.